ParisBNF/abb/BNFabb18.xml

```diff
@@ -0,0 +1,156 @@
+                           <dimensions type="outer">
+                              <height unit="mm" cert="medium">200</height>
+                              <width unit="mm" cert="medium">198</width>
+                           </dimensions>
+                        </extent>
+                     </supportDesc>
+
+
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="23">                               
+                        </layout>
+
+
+                     </layoutDesc>
+                     
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">19th-century script</seg>                        
+                     </handNote>
+                  </handDesc>                  
+               </physDesc>
+               <history>
+                  <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                        <date>1902</date>.</acquisition>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                              <bibl>
+                                 <ptr target="bm:ContiRossini1914Notice"/>
+                                 <citedRange unit="page">142</citedRange>
+                                 <citedRange unit="number">116</citedRange>
+                              </bibl>
+                              <bibl>
+                                 <ptr target="bm:Chaine1912Catalogue"/>
+                                 <citedRange unit="page">12-13</citedRange>
+                                 <citedRange unit="number">18</citedRange>
+                              </bibl>
+                              <bibl>
+                                 <ptr target="bm:Abbadie1859Cat"/>
+                                 <citedRange unit="page">19-21</citedRange>
+                                 <citedRange unit="number">18</citedRange>
+                              </bibl>
+                           </listBibl>
+                        </source>
+                     </recordHist>
+
+                  </adminInfo>
+
+
+               </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>               
```

Comment:
I didn't know about listPerson! It's good to learn something new

Reply:
Denis is to be thanked for it!

Reply:
It is not necessary to list the persons if they are quoted elsewhere with the relevant roles, but it is helpful if there is no other place to mention them, or if the context did not allow to specify each role